CONTRIBUTING.md

```diff
@@ -122,19 +144,18 @@ See guidelines for [contribution with CSS](https://github.com/liferay/clay/blob/
 
 ### Documentation Style Guide
 
--   Use [JSDOC](http://usejsdoc.org/) for documentation in your JavaScript code.
-
-#### Example
-
-```javascript
-/**
- * Data to add to the element.
- * @default undefined
- * @instance
- * @memberof ClayActionsDropdown
- * @type {?object}
- */
-data: Config.object();
+Clay uses [TypeScript](https://www.typescriptlang.org/) for typed, thereby overwriting [JSDOC](http://usejsdoc.org/) writing but adding descriptions to `interface`, `type`, `enum`, etc... to make it easier to use components.
```

Comment:
This reads a bit strangely. I suggest getting rid of "for typed", and saying something like:

> Clay uses TypeScript, obviating the need to write explicit JSDoc annotations for parameter and return types. We still write descriptions, however, for `interface`, `type`, `enum` etc to make it easier to use components.